docs/source/app-dev/index.rst

```diff
@@ -0,0 +1,80 @@
+
+  :doc:`Read the documentation for the gRPC API </app-dev/grpc/index>`
+- **The application architecture guide**: this documentation gives high-level guidance on how to build your application.
+
+  :doc:`Read the application architecture guide </app-dev/app-arch>`
+
+What's in the Ledger API
+************************
+
+No matter how you're accessing it (Java bindings, node.js bindings, or gRPC), the Ledger API exposes the same services:
+
+- Submitting commands to the ledger
+
+  - Use the **command submission service** to submit commands (create a contract or exercise a choice) to the ledger.
+  - Use the **command completion service** to track the status of submitted commands.
+  - Use the **command service** for a convenient service that wraps both of the above.
+- Reading from the ledger
+
+  - Use the **transaction service** to retrieve transactions of events (contracts created and contracts archived) from the ledger.
+  - Use the **active contract service** to quickly bootstrap an application with active contracts. It means you don't need to read from the beginning of the ledger or process create events for contracts that have already been archived.
+- Utility services
+
+  - Use the **package service** to query the DAML packages deployed to the ledger.
+  - Use the **ledger identity service** to retrieve the Ledger ID of the ledger the application is connected to.
+  - Use the **ledger configuration service** to retrieve some dynamic properties of the ledger, like minimum and maximum TTL for commands.
+  - Use the **time service** to obtain the time as known by the ledger server.
+
+For full information on the services see :doc:`/app-dev/services`.
+
+You may also want to read the :doc:`protobuf documentation <grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
+
+.. _daml-lf-intro:
+
+DAML-LF
+*******
+
+When you :ref:`compile DAML source into a .dar file <assistant-manual-building-dars>`, the underlying format is DAML-LF. DAML-LF is similar to DAML, but is stripped down to a core set of features. The relationship between the surface DAML syntax and DAML-LF is similar to that between Java and JVM bytecode.
+
+As a user, you don't need to interact with DAML-LF directly. But inside the DAML SDK, it's used for:
+
+- Executing DAML code on the Sandbox or on another platform
+- Sending and receiving values via the Ledger API (using a protocol such as gRPC)
+- Generating code in other languages for interacting with DAML models (often called “codegen”)
+
+.. DAML-LF content appears in the package service interactions. It is represented as opaque blobs that require a secondary decoding phase.
+
+When you need to know about DAML-LF
+===================================
+
+Knowledge of DAML-LF can be helpful when using the Ledger API or bindings on top of it. Development is easier if you know what the types in your DAML code look like at the DAML-LF level.
```

Comment:
We are trying to hide DAML-LF in contexts other than direct interaction with the API. As soon as you use codegen you don't need to know LF. Could we qualify this further in that direction? I'd hate for users to feel like they need to get their heads around DAML-LF. Not even sure I like it being mentioned on the landing page.

Reply:
No, I see what Bernhard means - I think I'd like to make it more explicit